src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/CSharpFormattingOptions2.cs

```diff
@@ -350,6 +351,9 @@ private static Option2<bool> CreateNewLineForBracesOption(NewLineOption newLineO
         public static Option2<bool> NewLinesForBracesInAccessors { get; } = CreateNewLineForBracesOption(
             NewLineOption.Accessors, nameof(NewLinesForBracesInAccessors));
 
+        public static Option2<bool> NewLinesForBracesInPropertyPatternClauses { get; } = CreateNewLineForBracesOption(
```

Comment:
@333fred FYI in case this needs change in O# as well:

https://github.com/OmniSharp/omnisharp-roslyn/blob/c9a8563c36acc6dfba6b28e1491d1267593c0732/src/OmniSharp.Shared/Options/FormattingOptions.cs#L138-L166